lms/models/_application_settings.py

```diff
@@ -0,0 +1,51 @@
+"""Model classes representing application settings."""
+
+
+class ApplicationSettings:
```

Comment:
This is very boring right now and doesn't have any business logic. But if we were going to implement any, this would be where we would.

For example, if we wanted different defaults, or reading from a "defaults" group or something.

We could also easily extend this with properties for the expected groups so settings could be accessed like:

```settings.canvas.sections_enabled```